prow/cmd/deck/main.go

```diff
@@ -102,6 +102,7 @@ const (
 type options struct {
 	configPath            string
 	jobConfigPath         string
+	common                prowflagutil.CommonOptions
```

Comment:
Also I am not sure people like generic names like `common`... maybe `port` or something related to their binding.
What do you think @alvaroaleman ?

Reply:
Yeah, something like `InstrumentationOptions` would be good, the current name is too common :)

Also I am wondering if it would make sense to make this part of the config file rather than a flag or is the use case local development?

Reply:
Use case is local development. I think it won't be used when Deck runs as Pod

Reply:
I'll rename it to InstrumentationOptions